GTG/gtk/editor/editor.py

```diff
@@ -536,14 +614,28 @@ def inserttag_clicked(self, widget):
             self.textview.insert_text("@", itera)
         else:
             self.textview.insert_text(" @", itera)
-        self.textview.grab_focus()
+        # self.textview.grab_focus()
```

Comment:
Please, remove this method, as you don't use it any more (you have the new fancy popoup don't you?)